src/renderers/dom/__tests__/ReactDOMProduction-test.js

```diff
@@ -178,6 +178,9 @@ describe('ReactDOMProduction', () => {
     expect(function() {
       class Component extends React.Component {
         render() {
+          // FIXME: undefined is a valid return type in Fiber, at least in the
+          // current implementation. It's treated as empty. I don't know why we
+          // sometimes get false positives.
```

Comment:
Maybe the call to `jest.resetModuleRegistry` kills the mock set up by test-framework-setup? If so then more tests could be incorrectly passing.